docs/modules/ROOT/pages/how-tos/configuring/customizing-the-build.adoc

```diff
@@ -1 +1,168 @@
-= Customizing the build pipeline
\ No newline at end of file
+= Customizing the build pipeline
+
+To meet your specific needs, you can customize the way that {ProductName} builds components. You can customize builds in two ways:
+
+* Change the parameters of the build pipeline of a component.
+* Extend the pipeline with your own tasks (or delete existing tasks).
+
+For example, you can change for how long {ProductName} saves the container images of component, by changing the value of the `image-expires-after` parameter. Or you can ensure that the container images it builds for a component are compliant with regulations for your industry, by adding a compliance check as a task to its build pipeline. 
+
+
+.Procedure
+
+To customize the build pipeline in any way, start by taking the following steps:
+
+. In your preferred IDE, navigate to the `.tekton` directory in the repository of your component.
+
+. Open the `testrepo-pull-request.yaml` file in the repository.
```

Comment:
I would add somewhere that if you also want for the task to be ran when merging to main as well, you need to do the same changes in the `testrepo-on-push.yaml` file

Reply:
I think that this does not include `push` because the procedure is explicitly talking about modifying the `image-expires-after` parameter. The page starts to be worded as a generic topic and then quickly narrows scope.

Reply:
Makes sense for the example, but I was referring that we need to tell users somewhere that if they need something to work in the two pipelines, they need to edit the two pipelines. Not everyone has worked with Tekton, and even less people with pipelines as code.

Reply:
I agree

Reply:
I'll add in the info that @ernesgonzalez33 is talking about, two separate YAMLs and you need to edit both for changes to be global. But just FYI, the current template repo for (upstream) Konflux apps only has a YAML for pull requests: https://github.com/konflux-ci/testrepo/blob/main/.tekton

Reply:
Odd. @yftacherzog, was that intentional?

Reply:
We're still working our way to make a fully working example for on-pr. I suppose that on-push example could come next, but it's going to take some time before we get there.